onnx/defs/gen_doc.py

```diff
@@ -371,10 +375,5 @@ def main(args):  # type: (Type[Args]) -> None
 
 
 if __name__ == '__main__':
-    base_dir = os.path.dirname(os.path.dirname(os.path.dirname(os.path.realpath(__file__))))
-    docs_dir = os.path.join(base_dir, 'docs')
-
-    class Args(object):
-        output = os.path.join(docs_dir, 'Operators' + ext)
-        changelog = os.path.join(docs_dir, 'Changelog' + ext)
-    main(Args)
+    gen_doc(is_gen_ml_doc=False)
+    gen_doc(is_gen_ml_doc=True)
```

Comment:
Currently the CIs failed with ONNX_ML=0 because ONNX_ML won't create ONNX_ML schema and the produced Operator-ml.md and Changelog-ml.md would be basically empty. 

Therefore I would say the behaviour of gen_doc.py should be as follows:
1. If ONNX_ML=1, it will update both general and *-ml.md documents
2. if ONNX_ML=0, it will only update general document

Then, it should help resolving the current CI failures. Thank you!